crates/meroctl/src/cli/context/grant.rs

```diff
@@ -0,0 +1,67 @@
+use calimero_context_config::types::Capability;
+use calimero_primitives::alias::Alias;
+use calimero_primitives::context::ContextId;
+use calimero_primitives::identity::PublicKey;
+use calimero_server_primitives::admin::{GrantPermissionRequest, GrantPermissionResponse};
+use clap::Parser;
+use eyre::OptionExt;
+use reqwest::Client;
+
+use crate::cli::Environment;
+use crate::common::{
+    fetch_multiaddr, load_config, make_request, multiaddr_to_url, resolve_alias, RequestType,
+};
+
+#[derive(Debug, Parser)]
+pub struct GrantPermissionCommand {
+    #[arg(help = "The context ID")]
+    #[arg(long, short, default_value = "default")]
+    pub context: Alias<ContextId>,
+
+    #[arg(help = "The granter's public key")]
+    #[arg(long, short = 'a', default_value = "default")]
+    pub granter: Alias<PublicKey>,
+
+    #[arg(help = "The grantee's public key")]
+    pub grantee: PublicKey,
+
+    #[arg(help = "The capability to grant")]
+    pub capability: Capability,
```

Comment:
define a local enum instead of the one in `context/config`

you can then annotate that enum with `clap`'s `ValueEnum`